R/autoplot.anosim.R

```diff
@@ -0,0 +1,44 @@
+##' @title ggplot-based plot for objects of class \code{"anosim"}
+##'
+##' @description
+##' Produces a multi-layer ggplot object representing the output of objects produced by \code{\link[vegan]{anosim}}.
+##'
+##' @param object an object of class \code{"anosim"}, the result of a call to \code{\link[vegan]{anosim}}.
+##' @param notch logical; make notched (default) or standard box plot?
+##' @param varwidth logical; make box width proportional to to the square-roots of the number of observations in the groups (default) or standart box plot?
+##' @param xlab character; label for the x-axis.
+##' @param ylab character; label for the y-axis.
+##' @param subtitle logical; should subtitle with R and p values be added.
+##' @param ... additional arguments passed to other methods.
+##' @return A ggplot object.
+##' @author Didzis Elferts
+##'
+##' @export
+##'
+##' @importFrom ggplot2 ggplot autoplot geom_boxplot aes_string labs fortify
+##'
+##' @examples
+##'
+##' data(dune)
+##' data(dune.env)
+##' dune.dist <- vegdist(dune)
+##' attach(dune.env)
+##' dune.ano <- anosim(dune.dist, Management)
+##'
+##' autoplot(dune.ano)
+
+`autoplot.anosim` <- function(object, notch = TRUE, varwidth = TRUE, subtitle = TRUE,
+                              xlab = NULL, ylab = NULL, ...) {
```

Comment:
To match the most recent additions to **ggvegan**, include a `title` argument to complement `xlab`, `ylab`. See for example [`autoplot.isomap`](https://github.com/gavinsimpson/ggvegan/blob/master/R/autoplot.isomap.R#L37) which also provides a default value for `title`. (Or use `NULL`, which is what `autoplot.envfit()` does.

Although I haven't added this yet, we're missing `subtitle` and `caption` as options, which I might sondeir as being required for `autoplot` methods for consistency. Thoughts?

Reply:
Arguments `title`, subtitle and caption added to the function. I agree that the same should be done with other functions.